code/game/objects/items/weapons/scissors.dm

```diff
@@ -84,11 +84,12 @@
 				user.visible_message("<span class='notice'>[user] stops cutting [M]'s hair.</span>", "<span class='notice'>You stop cutting [M]'s hair.</span>")
 				return
 			if(f_new_style)
-				H.f_style = f_new_style
+				C.f_style = f_new_style
 			if(h_new_style)
-				H.h_style = h_new_style
+				C.h_style = h_new_style
 
 		H.update_hair()
+		H.update_fhair()
```

Comment:
Fixes a bug where scissors wouldn't update facial hair after you gave a guy a trim.